api/core/Directus/Bootstrap.php

```diff
@@ -1082,4 +1121,24 @@ private static function socialAuth()
     {
         return new Social();
     }
+
+    private static function cache()
+    {
+//        $client = new \Memcache();
+//        $client->connect('localhost', 11211);
+//        $pool = new MemcacheCachePool($client);
+
+        $filesystemAdapter = new Local(__DIR__.'/../../../cache/');
```

Comment:
Can you make this default to `storage`, and let the user set this value to anything they want?